packages/arviz-base/pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[project]
+name = "arviz-base"
+version = "0.1.0"
```

Comment:
This should be 0.4.0 or maybe 0.5.0 (as we are going to do a release after we have the metapackage working), same for the other subpackaes

Reply:
what about dependencies? for instance arviz-base depends on  numpy and scipy